crates/holochain/src/core/workflow/validation_receipt_workflow/tests.rs

```diff
@@ -10,10 +9,8 @@ use holochain_keystore::AgentPubKeyExt;
 use holochain_state::prelude::*;
 use holochain_types::prelude::block::BlockTargetId;
 use holochain_types::prelude::*;
-use rusqlite::Transaction;
 
 #[tokio::test(flavor = "multi_thread")]
-#[ignore = "flaky"]
```

Comment:
I'm not certain that this isn't still flaky but it's a really important test and I haven't seen it flake on CI yet for this PR. If it does prove to be a problem I will disable it again and work on it.